src/routes/(site)/blog/[id]/+page.svelte

```diff
@@ -5,9 +5,23 @@
   import { Temporal } from '@js-temporal/polyfill';
   import Labels from '$lib/components/blog/labels.svelte';
   import BlogBody from './blog-body.svelte';
+  import { MetaTags } from 'svelte-meta-tags';
   export let data: PageData;
```

Comment:
Nit: Leave a space between the imports and the exports.
```suggestion
  import { MetaTags } from 'svelte-meta-tags';

  export let data: PageData;
```

Reply:
done